Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "HSLuvSwift",
+    platforms: [
+        .iOS(.v8),
+        .macOS(.v10_10),
```

Comment:
In the latest [podspec](https://github.com/hsluv/hsluv-swift/blob/master/HSLuvSwift.podspec) I see:

```
s.ios.deployment_target = '13.2'
s.osx.deployment_target = '10.11'
```

Reply:
I've fixed this in the newly created PR.